MakerTracker/DBModels/Transaction.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace MakerTracker.DBModels
+{
+    public class Transaction
+    {
+        public int Id { get; set;}
+        public Product Product { get; set; }
```

Comment:
The Product class - you can add a ICollection<Transaction> Transactions {get;set;} => new List<Transaction>() to that

Reply:
Like wise the Product class will have 2 Lists of Transactions - one that represents the ToAccounts and one that represents the FromAccounts(). You may have to specify a [ForeignKey()] Attribute so it knows which list points to which column in the transaction table.

Reply:
is the purpose behind this to just make it easier later to sum up all transactions for a given Product?